source/sky_triangle/skytriangle.cpp

```diff
@@ -173,6 +173,11 @@ void SkyTriangle::resize(int w, int h)
     m_height = h;
 }
 
+void SkyTriangle::resize(GLFWwindow * window)
+{
+    glfwGetFramebufferSize(window, &m_width, &m_height);
```

Comment:
So is there a way to trigger the framebuffer resize event manually to set the scale of the example with the framebuffer size on start? It is necessary to make sure the example is in the right scale before the first resizing.